products/BellHybrid/apps/application-bell-settings/models/LayoutModel.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#include <common/models/LayoutModel.hpp>
+
+#include <apps-common/ApplicationCommon.hpp>
+#include <db/SystemSettings.hpp>
+
+namespace app::bell_settings
+{
+
+    LayoutModel::LayoutModel(ApplicationCommon *app)
```

Comment:
```suggestion
    explicit LayoutModel::LayoutModel(ApplicationCommon *app)
```

Reply:
This constructor is declared explicit in the header file